### ISSUE TITLE
"no matching binding" error when running as separate forger

### ISSUE THREAD

**Issue by air1one** [ArkEcosystem/core#3912]
When running forger separate got his error : 
```
[2020-07-24 09:17:12.104] INFO : Round: 7,712, height: 393,286
1|v3-core-forger  | [2020-07-24 09:17:14.111] ERROR: Error: No matching bindings found for serviceIdentifier: Symbol(Service<Database>)
1|v3-core-forger  |     at _validateActiveBindingCount (/home/ark/core/node_modules/inversify/lib/planning/planner.js:62:23)
1|v3-core-forger  |     at _getActiveBindings (/home/ark/core/node_modules/inversify/lib/planning/planner.js:48:5)
1|v3-core-forger  |     at _createSubRequests (/home/ark/core/node_modules/inversify/lib/planning/planner.js:85:26)
1|v3-core-forger  |     at Object.plan (/home/ark/core/node_modules/inversify/lib/planning/planner.js:136:9)
1|v3-core-forger  |     at /home/ark/core/node_modules/inversify/lib/container/container.js:317:37
1|v3-core-forger  |     at Container._get (/home/ark/core/node_modules/inversify/lib/container/container.js:310:44)
1|v3-core-forger  |     at Container.get (/home/ark/core/node_modules/inversify/lib/container/container.js:230:21)
1|v3-core-forger  |     at Application.get (/home/ark/core/packages/core-kernel/dist/application.js:332:31)
1|v3-core-forger  |     at Object.exports.getBlockTimeLookup (/home/ark/core/packages/core-kernel/dist/utils/get-blocktime-lookup.js:27:33)
1|v3-core-forger  |     at ForgerService.checkSlot (/home/ark/core/packages/core-forger/dist/forger-service.js:128:85)
1|v3-core-forger  | [2020-07-24 09:17:14.112] INFO : Round: 7,712, height: 393,287
```

To reproduce, run separate forger process like so relay on its own `relay:run` and same for forger `forger:run`

**Comment by air1one** [comment]
Fixed by #3915

**Comment by air1one** [comment]
Was not fixed by #3915 in the end, re-opening.

**Comment by air1one** [comment]
Fixed by #3936
